diff --git a/out/util/appFileCopier.js b/out/util/appFileCopier.js
index 230f5bcd04697279ad8b43f406c9112637065c98..1d4986e85ffa63d3a829c15d598dd47e46e8bae6 100644
--- a/out/util/appFileCopier.js
+++ b/out/util/appFileCopier.js
@@ -215,10 +215,6 @@ async function resolveFirstMatchingCollection(options) {
         for (const dir of searchDirectories) {
             builder_util_1.log.info({ pm, searchDir: dir }, "searching for node modules");
             const deps = await run(pm, dir);
-            if (deps.nodeModules.length === 0) {
-                builder_util_1.log.info({ pm, searchDir: dir }, "no node modules found in collection, trying next search directory");
-                continue;
-            }
             if (collectionMatchesAppDependencies(deps.nodeModules, dependencies)) {
                 builder_util_1.log.debug({ pm, searchDir: dir, depCount: deps.nodeModules.length }, "collected node modules");
                 return deps;
